src/qaqc/01-organize_raw_data.Rmd

```diff
@@ -666,7 +666,7 @@ generate_weekly_flag_plots <- function(site_arg, parameter_arg, flag_arg = NULL)
       labs(x = "Datetime",
            y = "Mean")
 
-    plot_list[[paste(site_arg, parameter_arg, as.character(flag_day))]] <- week_plot
+    plot_list[[paste(site_param, as.character(flag_day))]] <- week_plot
```

Comment:
I am doing this so that it's easier to explore the data in the lists while we are developing this workflow. I like to look at what is happening at the data at different points. I believe that this specific change is just because I was making a new site-param object within this function so I just wanted to use that instead of what I had previously.